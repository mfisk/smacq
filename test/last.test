#!/bin/sh

$SMACQQ -g -m <<EOF | sort -n
print ts, srcip, dstip from (last srcip, dstip from pcapfile(0325@1112-snort.pcap))
EOF
