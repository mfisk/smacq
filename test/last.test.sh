#!/bin/sh

$SMACQQ -g -f - <<EOF | sort -n
print ts, srcip, dstip from (last srcip, dstip from pcapfile(0325@1112-snort.pcap))
EOF
