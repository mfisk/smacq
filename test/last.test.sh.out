2000-03-25T12:13:29.363329-07:00	192.168.0.54	192.168.0.210
2000-03-25T12:15:31.922436-07:00	192.168.0.200	192.168.0.54
2000-03-25T12:15:53.354234-07:00	192.168.0.54	192.168.0.200
